docs/source/overview/running.rst

```diff
@@ -54,6 +57,79 @@ Note that you can also convolute a single grid with a single eko (obtaining a si
 If necessary it is possible to specify the values of the *renormalization* and *factorization* scale variations with
 the options ``--xir`` and ``--xif``.
 
+
+Generating FONLL FK Tables
+--------------------------
+
+In order to generate FK tables with (numerical) |FONLL| the different flavor schemes
+need to be evolved separately and joined together in a single FK table only
+in the final step.
+
+There are 2 workflows possible, one in which all the steps are performed individually
+and one that is more automatized.
+
+The automatized workflows always assumes that |FONLL| is performed both for
+charm and bottom effects.
+
+The manual procedure works as follow:
+
+  1. Generate 7 theories for all the different flavor patches with command::
+
+      pineko fonll_tcards THEORY_ID
+
+    The different flavor patches are named following the convention:
+
+      * ``THEORY_ID00`` : |FFNS| :math:`n_f=3`
+      * ``THEORY_ID01`` : |FFN0| :math:`n_f=3`
+      * ``THEORY_ID02`` : massless component of |FFNS| :math:`n_f=4`
+      * ``THEORY_ID03`` : massive component of |FFNS| :math:`n_f=4`
+      * ``THEORY_ID05`` : massless component of |FFNS| :math:`n_f=5`
+      * ``THEORY_ID06`` : massive component of |FFNS| :math:`n_f=5`
+
+    where for |FFNS| :math:`n_f=4,5` massive and massless parts are splitted to
+    allow for a damping option.
+
+  2. Generate the grids corrisponding to all the 7 theories with the external program.
```

Comment:
Hmm this is a bit ugly. I understand wanting to keep things modular, but I think we should really have a step-by-step guide somewhere that explains how to use pinefarm/yadism together with pineko to create nFONLL FKtables. We will put it in the paper, but it would also be good to have it in a living document, i.e. docs, somewhere.

Reply:
True, but this is why I wanted a single place where you can run everything...
As far as `pineko` is concerned we can add a link to `pinefarm` of `pineline` (although the page would be non existing yet.)